okhttp/src/main/java/io/grpc/okhttp/OkHttpServerTransport.java

```diff
@@ -289,6 +294,10 @@ private void triggerGracefulSecondGoaway() {
       } else {
         frameWriter.flush();
       }
+      if (gracefulShutdownPeriod != null) {
+        forcefulCloseTimer = scheduledExecutorService.schedule(
+            this::triggerForcefulClose, gracefulShutdownPeriod, TimeUnit.NANOSECONDS);
```

Comment:
This changes the behavior when someone is doing:

```java
server.shutdown();
server.awaitTerminated(10000, TimeUnit.SECONDS);
```

Reply:
I forgot to mention that https://github.com/grpc/grpc-java/pull/9649 changed the behavior of this. So I put it back.
So sound like we should do graceful shutdown when `server.shutdown`?

Reply:
never mind about #9649.
I updated `server.shutdown()` to avoid forceful close.